data/hai/hai reveal 1 intro.txt

```diff
@@ -11,47 +11,560 @@
+			`	As you arrive you are greeted by a message requesting that you report to the operations center inside the main promenade. You proceed to equip your suit and exit your ship. The station itself is well preserved, except for the lack of atmosphere. Nothing seems to have vented it, no big hole in the ceiling, and the expansive ceiling windows of the main promenade provide a crystal clear view of the station's many decks towering above you. Exterior navigation lights are gradually flicking on up the height of the station, so that walking through feels like watching a ghost city slowly come to glittering life. Emergency floor lighting has been activated, showing you the way.`
+			`	Once through the operation center's security airlock you are able to remove your helmet. One Hai who appears to be in charge swiftly approaches from the other side. "Captain <last>! It is splendid you could make it here so quickly. We have begun the re-powering process, but it will take time and supplies to re-pressurize more than the docking and operations areas. These stations are not small."`
+			`	He chuckles as if having made a joke, as if you hadn't just walked beneath the open view of over three hundred decks above you. He seems oblivious to the strangeness of this, so perhaps it is a Hai thing.`
+			`	"I am sure you have questions but they will have to wait until we're more established. We have supplies coming and they will be vulnerable to any pirates that may poke their noses in. We need you to get out there and ensure the supplies can all arrive here safely. Local security is still about, so it shouldn't be too problematic."`
+			choice
+				`	(Go immediately.)`
+					accept
+				`	(Loiter to find out more.)`
+			`	He blinks at you. "Well, what are you waiting for? Go! Come back when all the supply ships have arrived safely."`
+			`	You are left with little choice other than to nod and get on your way.`
+				accept
+	on accept
+		set "hr: sandsnap project"
+	on visit
+		dialog "The supply ships have not all arrived safely yet."
+	on fail
+		dialog "The supply convoy has been lost."
+	npc accompany
+		government "Hai Merchant"
+		personality escort timid
+		system "Ula Mon"
+		fleet "Large Hai Merchant" 3
+	on complete
+		"reputation: Hai" += 200
+		"reputation: Hai (Wormhole Access)" += 200
+		"reputation: Hai Merchant" += 200
+		"reputation: Hai Merchant (Human)" += 200
+		"reputation: Hai Merchant (Sympathizers)" += 200
+		event "hr: sandsnap awoken" 5
+		conversation
+			`Peeps are all set up, Sandsnap will help out a little.`
+			While suiting up, receive message to not bother. Thanks (and payment?) but still no time for answers.
+
+
+event "hr: sandsnap awoken"
+	planet Sandsnap
+		government "Hai"
+		bribe 0.5
+		security 1
+		music ambient/machinery
+		description "This station was recently repowered and is now used as a military base where Hai warships can resupply and repair. There is a constant presence here ever since the pirate raids on Allhome."
```

Comment:
```suggestion
		description "This station was recently repowered and is now used as a military base where Hai warships can resupply and repair. There has been a constant presence here ever since the pirate raids on Allhome."
```